packages/serverless/src/backend_connector.js

```diff
@@ -223,6 +224,14 @@ function send(resourcePath, payload, finalLambdaRequest, callback) {
   req.on('timeout', () => onTimeout(req, resourcePath, payload, finalLambdaRequest, callback));
 
   req.on('error', e => {
+    // CASE: we manually destroy streams, skip these errors
+    // Otherwise we will produce `Error: socket hang up` errors in the logs
+    // We already print the warning that a timeout happened
+    // https://nodejs.org/api/http.html#requestdestroyed
+    if (req.destroyed) {
```

Comment:
When a timeout occurs when talking to the layer, we destroy the initial request, which results in the error `socket hang up` in the logs. In parallel we send the request directly. 

The error itself should be ignored, otherwise it's a confusing error and the data would be sent twice to the BE directly.

The second problem in the ticket was that the lambda kept alive 120s (the configured timeout). I doubt this is related to the fix, but I don't know for sure. Might be caused by the customer. We'll see.